build/common.ps1

```diff
@@ -313,7 +313,8 @@ Function New-Package {
 		[string]$ReleaseLabel = "zlocal",
 		[string]$BuildNumber,
 		[switch]$NoPackageAnalysis,
-		[string]$Version
+		[string]$Version,
```

Comment:
The simple blob download seems extremely easy, with the reference `common.ps1` being the one in ServerCommon. I'm going to try implementing that.